web_external/js/views/body/ImagesView/ImagesFacetsPane.js

```diff
@@ -0,0 +1,48 @@
+isic.views.ImagesFacetsPane = isic.View.extend({
+    render: function () {
+        if (!this.addedCollapseImage) {
```

Comment:
I agree. We kept it around in this PR just out of caution, and resolved to cut out unneeded assets and associated code in a separate sweep.